GUI/SplitContainerAdv.cs

```diff
@@ -1,123 +1,150 @@
-  file, You can obtain one at http://mozilla.org/MPL/2.0/.
- 
-  Copyright (C) 2009-2010 Michael Möller <mmoeller@openhardwaremonitor.org>
-	
-*/
+﻿// Mozilla Public License 2.0
+// If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+// Copyright (C) LibreHardwareMonitor and Contributors
+// All Rights Reserved
 
 using System;
 using System.Collections.Generic;
 using System.Drawing;
 using System.Windows.Forms;
 
-namespace OpenHardwareMonitor.GUI {
-  public class SplitContainerAdv : SplitContainer {
+namespace OpenHardwareMonitor.GUI
+{
+    public class SplitContainerAdv : SplitContainer
+    {
+        private int _delta = 0;
+        private Border3DStyle _border3DStyle = Border3DStyle.Raised;
+        private Color _bgColor = SystemColors.Control;
 
-    private int delta = 0;
-    private Border3DStyle border3DStyle = Border3DStyle.Raised;
-    private Color color = SystemColors.Control;
-
-    public SplitContainerAdv()
-      : base() {
-      SetStyle(ControlStyles.ResizeRedraw, true);
-      SetStyle(ControlStyles.AllPaintingInWmPaint, true);
-      SetStyle(ControlStyles.UserPaint, true);
-      SetStyle(ControlStyles.OptimizedDoubleBuffer, true);
-      SetStyle(ControlStyles.ContainerControl, true);
-      UpdateStyles();
-    }
-
-    protected override void OnPaint(PaintEventArgs e) {
-      base.OnPaint(e);
+        public SplitContainerAdv()
+          : base()
+        {
+            SetStyle(ControlStyles.ResizeRedraw, true);
+            SetStyle(ControlStyles.AllPaintingInWmPaint, true);
+            SetStyle(ControlStyles.UserPaint, true);
+            SetStyle(ControlStyles.OptimizedDoubleBuffer, true);
+            SetStyle(ControlStyles.ContainerControl, true);
+            UpdateStyles();
+        }
 
-      Graphics g = e.Graphics;
-      Rectangle r = SplitterRectangle;
-      using (SolidBrush brush = new SolidBrush(color))
-        g.FillRectangle(brush, r);
-      ControlPaint.DrawBorder3D(g, r, border3DStyle);
-    }
+        protected override void OnPaint(PaintEventArgs e)
+        {
+            base.OnPaint(e);
+            Graphics g = e.Graphics;
+            Rectangle r = SplitterRectangle;
+            using (SolidBrush brush = new SolidBrush(_bgColor))
+                g.FillRectangle(brush, r);
+            ControlPaint.DrawBorder3D(g, r, _border3DStyle);
+        }
 
-    protected override void OnKeyDown(KeyEventArgs e) {
-      if (!base.IsSplitterFixed) {
-        if (e.KeyData == Keys.Right || e.KeyData == Keys.Down) {
-          SplitterDistance += SplitterIncrement;
-        } else if (e.KeyData == Keys.Left || e.KeyData == Keys.Up) {
-          SplitterDistance -= SplitterIncrement;
+        protected override void OnKeyDown(KeyEventArgs e)
+        {
+            if (!base.IsSplitterFixed)
+            {
+                if (e.KeyData == Keys.Right || e.KeyData == Keys.Down)
+                {

```

Comment:
Do you know if there's an editor config to remove braces in a single line if/else? As long as a block is properly indented I think it's unnecessary to have these braces.

Reply:
Sometimes it is an good idea to have braces. (multiline conditions, or long if statements)
this make the code more readable

Reply:
In some cases, but in most cases - like this one - it's not an improvement I think. I've been looking into it and you can set csharp_prefer_braces to when_multiline. Then as a suggestion instead of error should be OK.